projects/example-app/src/app/core/containers/app.component.ts

```diff
@@ -12,10 +11,10 @@ import { LayoutActions } from '@example-app/core/actions';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <bc-layout>
-      <bc-sidenav [open]="(showSidenav$ | async)!" (closeMenu)="closeSidenav()">
+      <bc-sidenav [open]="showSidenav()!" (closeMenu)="closeSidenav()">
```

Comment:
We shouldn't need these anymore:

```suggestion
      <bc-sidenav [open]="showSidenav()" (closeMenu)="closeSidenav()">
```